lib/compiler.js

```diff
@@ -65,6 +65,13 @@ exports.compile = function (code, options) {
     });
 
     importExportVisitor.finalizeHoisting();
+
+    if (getOption(options, "wrapWithRun")) {
+      result.code = "module.run(function(){" +
+        magicString.toString() + "//*/\n});";
+
+      return result;
+    }
```

Comment:
The only reason `wrapWithRun` exists is because of the REPL so I don't think the option should exist. Maybe instead `run` can detect it's in a REPL or maybe the repl could add its own `run` method that doesn't set the `this`.